lib/ruby_lsp/executor.rb

```diff
@@ -25,6 +25,7 @@ def execute(request)
       request_time = Benchmark.realtime do
         response = run(request)
       rescue StandardError, LoadError => e
+        raise e if $RUBY_LSP_TEST
```

Comment:
Wasn't sure if there was a more 'standard' way to name such a thing.

Reply:
I think it is better to make this just:
```suggestion
        raise if $RUBY_LSP_TEST
```
so that the original source of the exception is kept, and it doesn't look like the exception was raised here. Ref: https://www.exceptionalcreatures.com/guides/advanced-rescue-and-raise#reraising-exceptions